src/Accounts/Accounts/Utilities/CommandMappings.json

```diff
@@ -364,6 +364,16 @@
       "Update-AzApplicationInsightsWorkbook": {},
       "Update-AzApplicationInsightsWorkbookTemplate": {}
     },
+    "Az.ArcResourceBridge": {
```

Comment:
Looks like the preview branch is behind main for a lot because this file shouldn;'t be updated. But it doesn't matter.